shimmer/develop/what-is-shimmer/clients.md

```diff
@@ -0,0 +1,105 @@
+---
+description: "You can use the IOTA client libraries to handle communications between your node and
```

Comment:
Just a small thing, but we don't use " anywhere else, no? Should we remove it here to?